### DISCUSSION TITLE
How to migrate baseId in ariakit/react?

### DISCUSSION THREAD

**Discussion by yeion7** [ariakit/ariakit#2585] [Questions] [answered]
In Reakit we had an option to set a base for all id items, `baseId` https://reakit.io/docs/tooltip/#usetooltipstate

I'm wondering if that options exist in ariakit?

<img width="424" alt="Screenshot 2023-06-26 at 11 06 53 AM" src="https://github.com/ariakit/ariakit/assets/9127504/4c84e888-5d88-4ddc-b17a-28dc9f489847">

**Comment by diegohaz** [comment, marked as answer]
You don't need `baseId` anymore. But you can set the `id` prop directly on the elements if you want.
